### DISCUSSION TITLE
1.21.1 version?

### DISCUSSION THREAD

**Discussion by drbleblo** [IntelligenceModding/AdvancedPeripherals#663] [Ideas]
I noticed you updated the pack yesterday but still no version for 1.21.1 is this in the works? if so how long I desperately need this mod to connect CC  to integrated dynamics

**Comment by SirEndii** [comment]
I am a bit confused 
The main reason why AP is not ported to a more modern version of mc is that I currently don't have a lot of time
So integrated dynamics or cc does not really have something to do with that

**Reply by drbleblo** [reply]
AP allows integrated to read NBT from a block I forget the process I used to do it but AP is the only way i've found to keep the process in game

**Comment by SL8RGaming** [comment]
Waiting for this as well setup a defragmenter system for AE2 just to realize I did not have this mod to get it working correctly just to find its not on 1.21.x yet I did see you are working on getting a 1.21 build and am grateful you have been able to do this this mod is amazing!

**Reply by SirEndii** [reply]
Hey
We currently work on a Port, keep an eye on #679 for progress and join our [discord](https://discord.intelligence-modding.de/) if you want to test AP 1.21.1 in the near future
